### ISSUE TITLE
Hydrogen Raindrop Explained in layman terms

### ISSUE THREAD

**Issue by iCollinberg** [HydroBlockchain/hcdp#146]
https://steemit.com/cryptocurrency/@collinberg/hydrogen-raindrop-easing-public-blockchain-authentication

Eth Wallet Address- 0x0a110a74440C44A21A833e121c89d5599f60d134

**Comment by ghost** [comment]
Thank you for your submission! We appreciate your involvement with the Hydro ecosystem.

Unfortunately, this post doesn't meet the standards of the HCDP; it does not provide an original, detailed, informative view of Hydrogen products. We’re currently looking for posts that add an additional layer of uniqueness beyond summarizing our current white paper or existing product offerings.
